app/src/main/java/com/example/keniphhomemaintenance/Dimens.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.keniphhomemaintenance
+
+object Dimens {
+
+    const val ADD_MAINTENANCE_ITEM_BUTTON_SIZE = 50
```

Comment:
Can you move the .dp into the const val as well? The variable represents a size in dp; we wouldn't use this constant for things other than specifying the size in dp, so we might as well bundle it into the value.